File: JAVA/laboratorio-um/src/PorcentagemAcerto.java
import java.util.Scanner;

public class PorcentagemAcerto {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int respostas[] = new int[100];
        int gabarito[] = new int[100];

        int valor = scan.nextInt();

        int quantidade = 0;
        while (valor > 0) {
            respostas[quantidade] = valor;
            valor = scan.nextInt();
            quantidade++;
        }

        valor = scan.nextInt();
        int i = 0;
        while (valor > 0) {
            gabarito[i] = valor;
            valor = scan.nextInt();
            i++;
        }

        int acertos = 0;
        for (int j = 0; j < quantidade; j++) {
            if(respostas[j] == gabarito[j]) {
                acertos++;
            }
        }


        double porcentagem = ((double) acertos)/ (double) quantidade;

        System.out.printf("%.2f\n", porcentagem*100);

        scan.close();
    }
}

File: Kotlin/laboratorio-01/src/main/kotlin/PorcentagemAcerto.kt
fun main() {
    var valores = readln().split("-1")

    var respostas = valores[0].trim().split(" ")
    var gabarito = valores[1].trim().split(" ")

    var acertos = 0
    for (i in 0..respostas.size - 1) {
        acertos += if (respostas[i] == gabarito[i]) 1 else 0
    }

    var porcentagem: Double = acertos.toDouble()/respostas.size.toDouble()

    println(String.format("%.2f", porcentagem * 100.0))
}